DaoBuilder/dcntDAOConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { DecentDAOConfig } from "./types";
+
+export const decentDAOConfig: DecentDAOConfig = {
+  lockStart: Math.floor(Date.now() / 1000), // (Now) Start time of token lock
+  lockDuration: 60 * 60 * 24 * 365, // 1 year
+  snapshotURL: "https://snapshot.org/#/decentdao.eth",
+  initialSupply: "1000000",
+  votingPeriod: 86400, // Length of time (in blocks) that voting occurs.
```

Comment:
these defaults were done when we were using seconds instead of blocks, but never updated because for the tests it didn't matter, but this is 1 day in seconds, not block

ultimately we should do `7200 * {number of days}` here, to get the blocks, I think, but someone should check my calculations as well

Reply:
The same formatting doesn't happen here. this goes straight into the transaction as number of blocks

Reply:
I imagine these will all get updated when we decide on these final values. but this goes straight into the contract as blocks (I've updated for testing to like 5 blocks, a little low but works for now)

Reply:
This config should be one big TODO that gets reviewed and updated after merging this PR but before deployment, as there may be other people that need input into this.